### ISSUE TITLE
Current progress is not added to aria label of next button 

### ISSUE THREAD

**Issue by BennyAlex** [gilbarbara/react-joyride#1074]
## 🐛 Bug Report

When enabling the progress it will be shown inside the button but is not added to the aria label and therefore not announced to screenreaders.

It would also be better if the aria labels says something more descriptive like "Proceed with next step (Step 1 of 6)" 

Thanks for helping improving web accessibility:)

**Comment by gilbarbara** [comment]
Hey @BennyAlex 

Fixed in 2.9.0
